### PR TITLE
Add images to team section

### LINKED ISSUE #36: Add images in "Our team" section

### PR DESCRIPTION
**Pull request by eldracode**
Fixes: #36

### CODE REVIEW

**Comment by karandevtyagi** [comment]
@eldraco19 good work.Just reduce the space between heading and the content of page in every page which has excess space.This will make it good on phone as well.

Issue3

**Comment by eldracode** [comment]
@karandevtyagi done!

**Comment by karandevtyagi** [comment]
@eldraco19 Toggle to mobile devices in your dev tools and look at the extra spaces in complete website and remove them.Plus add credit to yourself in footer or add github repo link so that you can show this in future to someone or in resume if required.

**Comment by eldracode** [comment]
@karandevtyagi on Desktop it looks fine, on mobile I tried to reduce as much as I can without affecting other components.

**Old**
old

**New**
new

File: index.html [hunk 1]
```diff
@@ -46,11 +48,12 @@
...
                 <li class="active"><a href="index.html" style="font-weight: bold;">Home</a></li>
-                <li><a href="about.html" style="font-weight: bold;">About</a></li>
-                <li><a href="judges.html" style="font-weight: bold;">Judges</a></li>
-                <li><a href="tracks.html" style="font-weight: bold;">Tracks</a></li>
-                <li><a href="diversity.html" style="font-weight: bold;">Divesity and Inclusion</a></li>
-                <li><a href="venue.html" style="font-weight: bold;">Venue</a></li>
+                <li><a href="about.html" style="font-weight: bold;">About Us</a></li>
+                <li><a href="judges.html"style="font-weight: bold;">Judges</a></li>
+                <li><a href="tracks.html"style="font-weight: bold;">Tracks</a></li>
+                <li><a href="diversity.html"style="font-weight: bold;">Divesity and Inclusion</a></li>
```
Comment (rexagod) [inline index.html]:
```suggestion
                <li><a href="diversity.html"style="font-weight: bold;">Diversity and Inclusion</a></li>
```

**Review by rexagod — CHANGES_REQUESTED** [review]

**Comment by rexagod** [comment]
https://github.com/JIITODC/trident/blob/ec2f74e354b4e0d6fcd8d9af031f33850184180c/index.html#L525-L536

Colorlib -> JIITODC? WDYT?

File: about.html [hunk 2]
```diff
@@ -149,11 +153,13 @@ <h2 class="text-center">Our Team</h2>
...
             <div class="testimony text-center">
               <figure>
-                <img src="images/profile.jpeg" alt="Image" class="img-fluid">
+
+                <img src="images/pranshu.jpg" alt="Image" class="img-fluid">
               </figure>
               <blockquote>
-                <p class="author mb-0">Foo bar</p>
-                <p class="text-muted mb-4">Coordinator</p>
+                <p class="author mb-0">Pranshu Shrivastava</p>
```
Comment (rexagod) [inline about.html]:
```suggestion
                <p class="author mb-0">Pranshu Srivastava</p>
```

File: about.html [hunk 3]
```diff
@@ -149,11 +153,13 @@ <h2 class="text-center">Our Team</h2>
...
+                <p class="text-muted mb-4">Tech and Management Head</p>
```
Comment (rexagod) [inline about.html]:
```suggestion
                <p class="text-muted mb-4">Technology and Management Head</p>
```

**Review by rexagod — CHANGES_REQUESTED** [review]
PTAL

**Comment by karandevtyagi** [comment]
About Event in footer of judge and tracks page require change.

**Comment by rexagod** [comment]
ping @eldraco19 ==> https://github.com/JIITODC/trident/pull/37#issuecomment-600270043

**Comment by eldracode** [comment]
@karandevtyagi footer fixed in all pages with social links and quick links(they were missing in some pages)

@rexagod Colorlib->JODC done!

**Comment by karandevtyagi** [comment]
@eldraco19 JODC in footer still leads in colorlib.

File: venue.html [hunk 4]
```diff
@@ -242,7 +148,7 @@ <h2 class="footer-heading text-uppercase mb-4">Connect with Us</h2>
               <div class="border-top pt-5">
               <p>
               <!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
-              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >Colorlib</a>
+              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >JODC</a>
```
Comment (rexagod) [inline venue.html]:
```suggestion
              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://github.com/JIITODC" target="_blank" >JODC</a>
```

File: tracks.html [hunk 5]
```diff
@@ -204,7 +205,7 @@ <h2 class="footer-heading text-uppercase mb-4">Connect with Us</h2>
               <div class="border-top pt-5">
               <p>
               <!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
-              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >Colorlib</a>
+              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >JODC</a>
```
Comment (rexagod) [inline tracks.html]:
```suggestion
              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="hhttps://github.com/JIITODC" target="_blank" >JODC</a>
```

File: judges.html [hunk 6]
```diff
@@ -134,7 +146,7 @@ <h2 class="footer-heading text-uppercase mb-4">Connect with Us</h2>
               <div class="border-top pt-5">
               <p>
               <!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
-              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >Colorlib</a>
+              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >JODC</a>
```
Comment (rexagod) [inline judges.html]:
```suggestion
              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://github.com/JIITODC" target="_blank" >JODC</a>
```

File: diversity.html [hunk 7]
```diff
@@ -133,7 +134,7 @@ <h2>Onsite Resources</h2>
               <div class="border-top pt-5">
               <p>
               <!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
-              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >Colorlib</a>
+              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >JODC</a>
```
Comment (rexagod) [inline diversity.html]:
```suggestion
              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://github.com/JIITODC" target="_blank" >JODC</a>
```

File: code-of-conduct.html [hunk 8]
```diff
@@ -218,7 +220,7 @@ <h2>The less Quick version</h2>
               <div class="border-top pt-5">
               <p>
               <!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
-              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >Colorlib</a>
+              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >JODC</a>
```
Comment (rexagod) [inline code-of-conduct.html]:
```suggestion
              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://github.com/JIITODC" target="_blank" >JODC</a>
```

File: about.html [hunk 9]
```diff
@@ -218,7 +391,7 @@ <h2 class="text-center">Our Team</h2>
               <div class="border-top pt-5">
               <p>
               <!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
-              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >Colorlib</a>
+              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >JODC</a>
```
Comment (rexagod) [inline about.html]:
```suggestion
              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://github.com/JIITODC" target="_blank" >JODC</a>
```

**Review by rexagod — CHANGES_REQUESTED** [review]
Could you also remove the `colorlib` comments?

**Comment by Ankk98** [comment]
@rexagod @eldraco19 
Have a look here: [Colorlib Licence](https://colorlib.com/wp/licence/ )
"You are not allowed to remove the copyright credit from the footer."
So, add credits to both: colorlib and jodc.

**Comment by Ankk98** [comment]
@rexagod @karandevtyagi 
Suggestion: 
I don't think we need to make another discord server for this. Instead add a channel in existing server and use the same. This will be more convinient.

**Comment by Ankk98** [comment]
@eldraco19 @rexagod @karandevtyagi  @VARoDeK 
We need to add term and conditions also. Especially the one saying that all final rights remain with the organisers.

**Comment by Ankk98** [comment]
@eldraco19 
Registration button do not work: Add something saying registration will start soon.

File: about.html (hunk 9 above)
Comment (Ankk98) [inline about.html]:
```suggestion
              Copyright &copy; <script>document.write(new Date().getFullYear());</script> All rights reserved | Made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >Colorlib</a> and <a href="https://github.com/JIITODC" target="_blank" >JODC</a>
```

Comment (Ankk98) [reply about.html]: Something like this would be better.

**Review by Ankk98 — CHANGES_REQUESTED** [review]
Make changes along the line of suggestions.

**Review by koderjoker — CHANGES_REQUESTED** [review]
Suggestions: 
1. Use `align-content: flex-end` on the divs with the class "row" containing headings (About, Judges) to improve the awkward spacing issue
2. Deleting the "About" heading from the main page (index.html) improves the look of the website since it doesn't seem to be necessary, and it sticks out like a sore thumb compared to the other subheadings which have a white font and a red underline

**Comment by eldracode** [comment]
@Ankk98 added a popup in the registration button.
@rexagod Added both colorlib and JODC in the footer.

**Comment by karandevtyagi** [comment]
> Suggestions:
> 
> 1. Use `align-content: flex-end` on the divs with the class "row" containing headings (About, Judges) to improve the awkward spacing issue
> 2. Deleting the "About" heading from the main page (index.html) improves the look of the website since it doesn't seem to be necessary, and it sticks out like a sore thumb compared to the other subheadings which have a white font and a red underline

- [ ] Add t&c

@rexagod @eldraco19 I think after this we can Squash and merge this PR and can release Trident v1.0

**Comment by Ankk98** [comment]
@karandevtyagi
I think this would be v0.91 or something.
There are still some minor issues that we need to fix. I will add some of them as issues. Then we should be showing the website to others (beta test).
After that if we find it complete, then it should be v1(release).

**Comment by karandevtyagi** [comment]
PiedPiper got it.

**Comment by Ankk98** [comment]
@eldraco19 
Kindly fix the inconsistent spacing between where-timeline , register-about etc.

**Comment by Ankk98** [comment]
@eldraco19 @rexagod 
About page and diversity and inclusion page are sideways scrollable/horizontally which is undesired.
Kindly fix it.

**Comment by eldracode** [comment]
@Ankk98 horizontal scroll fixed

**Comment by eldracode** [comment]
resolves #43 
resolves #45  
resolves #46 
resolves  #40 
resolves  #38  
resolves #47

**Comment by rexagod** [comment]
There's a spacing issue between the map and footer on venue page. PTAL.

**Comment by Ankk98** [comment]
@eldraco19 
- [ ] Add a notice in the 'when' section that event has been postponed due to COVID-19 pandemic.
- [ ] Add how to reach jiit-128 through the cab(Like dest address me exactly kya type karne se aayega) and nearest metro stations of all the 3 lines in the venue page.

I think after this PR we can go for a beta test @karandevtyagi @rexagod.

**Comment by Ankurk99** [comment]
@Ankk98 on it.

**Comment by Ankurk99** [comment]
Done #49

**Comment by eldracode** [comment]
@rexagod spacing issue in venue page fixed

**Comment by eldracode** [comment]
I guess we should merge this now. I am leaving rest of the CSS issues for @JIITODC/trident-volunteers
I will add the TnC page in another PR.

**Review by karandevtyagi — APPROVED** [review]

**Review by Ankk98 — APPROVED** [review]

**Comment by rexagod** [comment]
@eldraco19 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥

**Pull request merged**